### ISSUE TITLE
Click on "Transfer successful" notification should open file or it's location with file selected

### ISSUE THREAD

**Issue by MichalLebeda** [andyholmes/gnome-shell-extension-gsconnect#1009]

**Comment by MichalLebeda** [comment]
Yeah good to know but still cannot use that notification from the list

**Comment by HaMF** [comment]
Hi guys, I got my hands dirty and tried implementing this. See #1039

Disclaimer: I'm neither a JS developer, nor used to GIO or experienced with dbus. In summary: I have no clue what I'm doing :monkey: 

I thought it's probably a small change, tried reading the relevant documentation and fiddled it together. I'm happy to improve the PR but need guidance. Otherwise it might still serve as starting point for a proper implementation. In particular, we can/should discuss 
* if dbus is really the only way to achieve this (that's what I found, should work for nautilus and dolphin)
* how tests can be added
* how I can check the transfer-failed notification works
* if we want to use this behavior when "Open Folder is clicked
